content/advanced-networking/secondary_cidr/eniconfig_crd.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Create CRDs"
+date: 2019-03-02T12:47:43-05:00
+weight: 40
+---
+
+### Create ENIConfig as a CRD
+As next step, we will define a new ENIConfig custom resource definition (CRD) for your cluster. CRD's are extensions of Kubernetes API that stores collection of API objects of certain kind. In this case, we will store VPC Subnet and SecurityGroup configuration information in these CRD's so that Worker nodes can access them to configure VPC CNI plugin.
+
+You should have ENIConfig CRD already installed with latest CNI version (1.3+). You can check if its installed by running this command.
+```
+kubectl get crd
+```
+If you don't have ENIConfig installed, you can create by using your favorite editor and paste the following
```

Comment:
I would omit the whole thing because on the page before you tell them to `kubectl apply -f https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/master/config/v1.3/aws-k8s-cni.yaml` if you don't want to omit tell them to run the `apply` command again on the cni manifest from the repo. My concern here is if we changed anything in the core CRD this _could_ overwrite those changes.

Reply:
fixed in latest commit